certificates/Certificate.crt

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Use `trustme` for certs, it's more robust approach.

Reply:
no problem